proto/reaction.proto

```diff
@@ -360,6 +360,7 @@ message Vessel {
     VesselPreparationType type = 1;
     string details = 2;
   }
+<<<<<<< HEAD
```

Comment:
@michaelmaser it looks like you've pushed conflicted merges. This is because I updated the `reaction.proto` code (and several other files) after your initial PR. The correct workflow in this case would be for you to `git pull` to update your version, make any modifications, then commit, test, and push. I assume that you started making your modifications before you pulled, and when you pushed you got a message saying your version is out of date. When you _then_ pulled, you might have gotten a warning that the merge could not be completed automatically. In this case, you need to carefully look at where the conflicts occur and manually resolve them. I'll fix all of these, but something to look out for in the future!

Reply:
That's exactly what happened. I had the proto open for editing not realizing there were changes made. Then thought I caught them after I got the error, but clearly didn't. Apologize for the confusion. Thanks.